test/Graphics/Image.test.cpp

```diff
@@ -425,6 +437,36 @@ TEST_CASE("[Graphics] sf::Image")
                 CHECK(image.saveToFile(filename));
             }
 
+            SECTION("To Spanish Latin1 filename .png")
+            {
+                // small n with tilde, from Spanish, outside of ASCII, inside common Latin 1 codepage
+                filename /= U"test-\u00f1.png";
+                CHECK(image.saveToFile(filename));
+            }
+
+            SECTION("To Polish filename .png")
+            {
+                // small n with acute accent, from Polish, outside of Latin 1 codepage
+                filename /= U"test-\u0144.png";
+                CHECK(image.saveToFile(filename));
+            }
+
+            SECTION("To Japanese CJK filename .png")
+            {
+                // CJK symbol for Sun, outside of any European language codepage
+                filename /= U"test-\u65E5.png";
+                CHECK(image.saveToFile(filename));
+            }
+
+            SECTION("To emoji non-BMP Unicode filename .png")
+            {
+                // snail emoji, outside of Unicode Basic Multilingual Plane
+                filename /= U"test-\U0001F40C.png";
+                CHECK(image.saveToFile(filename));
```

Comment:
Each section is ending with this line. It could as well be written once out of sections.

Reply:
@ChrisThrasher  will have to say on this. I have no opinion or knowledge what's better in catch2.

Reply:
Good idea. I implemented that in my recent change.